doc/sphinx/language/core/basic.rst

```diff
@@ -400,6 +400,20 @@ The :term:`flag`, :term:`option` and :term:`table`
 mechanisms are used to modify the behavior of Rocq more globally in a
 document or project.
 
+.. _good_defaults:
+
+The default value of some settings is not the one that would be recommended
+today for compatibility reasons. We provide a *good default* file in the Corelib
+which can be imported as follows:
+
+.. rocqtop::
+
+   From Corelib Require Import GoodDefaults_2025.
+
+This file is versioned to account for the changes in recommendations as the Rocq
```

Comment:
Do you expect these recommendations will over time become the default settings?  If changes in the defaults are manageable for users,  maybe that would be simpler.

Reply:
It would be great, but it is much more work, as it can break things on the user side. For now it's what we recommend for new users / files.